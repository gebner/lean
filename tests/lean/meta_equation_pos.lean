meta def f (x : nat) : nat :=
tt -- type error should be reported here

check nat
