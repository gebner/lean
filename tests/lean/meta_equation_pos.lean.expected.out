meta_equation_pos.lean:2:0: error: equation type mismatch, expression
  tt
has type
  bool
but is expected to have type
  ℕ
meta_equation_pos.lean:1:9: warning: declaration 'f' uses sorry
ℕ : Type
